js-exercises/iter-infinite/iterInfinite.js

```diff
@@ -0,0 +1,33 @@
+function* count(start, step) {
+  const jump = step || 1;
+  for (let i = start; i <= Infinity; i += jump) {
+    yield i;
+  }
+}
+
+function* cycle(iterable, times) {
+  let i = 0;
+  let loopCount = 0;
+  do {
+    yield iterable[i];
+    i += 1;
+    if (i === iterable.length) {
+      i = 0;
+      loopCount += 1;
+    }
+  } while (i < iterable.length && loopCount !== times);
```

Comment:
I don't particularly see the importance of the `i < iterable.length` condition, because it will never be `false` as per the implemented logic.